src/CommonITILObject.php

```diff
@@ -3313,7 +3313,7 @@ public static function dropdownStatus(array $options = [])
 
         switch ($p['showtype']) {
             case 'allowed':
-                $tab = static::getAllowedStatusArray($p['value_calculation'] ?? $p['value']);
+                $tab = static::getAllowedStatusArray($p['value_calculation'] ?: $p['value']);
```

Comment:
issue is, with elvis operator, you now can have a warning for undefined key if `value_calculation` is not set in `$p` array.

I'm not sure you can have a short syntax for this
You will need to have something like this:

```php 
isset($p['value_calculation']) && strlen($p['value_calculation']) > 0
   ? $p['value_calculation']
   : $p['value'];
```